docs/source/viewers.rst

```diff
@@ -0,0 +1,59 @@
+.. _viewers:
+
+**************************
+Live preview
```

Comment:
I'd suggest changing the terminology from "preview" to either "viewer" or "display" for consistency with the API

Reply:
Or "Image viewer" or "Image display"